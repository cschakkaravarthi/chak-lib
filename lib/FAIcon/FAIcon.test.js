"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
exports.__esModule = true;
var react_1 = __importDefault(require("react"));
var testHelper_1 = require("../services/testHelper");
var FAIcon_1 = __importDefault(require("./FAIcon"));
describe('FAIcon', function () {
    it('should FAIcon render correctly', function () {
        var tree = testHelper_1.shallowRender(react_1["default"].createElement(FAIcon_1["default"], { color: 'red', iconSize: 'sm', className: '', iconName: 'calendar' }));
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=FAIcon.test.js.map