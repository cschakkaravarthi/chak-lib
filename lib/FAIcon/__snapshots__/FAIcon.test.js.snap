// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAIcon should FAIcon render correctly 1`] = `
<FontAwesomeIcon
  border={false}
  className=""
  color="red"
  fixedWidth={false}
  flip={null}
  icon="calendar"
  inverse={false}
  listItem={false}
  mask={null}
  pull={null}
  pulse={false}
  rotation={null}
  size="sm"
  spin={false}
  swapOpacity={false}
  symbol={false}
  title=""
  transform={null}
/>
`;
